github-graphql/Cargo.toml

```
@@ -5,4 +5,7 @@ edition = "2021"
 
 [dependencies]
 chrono = { version = "0.4", features = ["serde"] }
-cynic = { version = "2.2.2" }
\ No newline at end of file
+cynic = { version = "3", features = ["rkyv"] }
```

```suggestion
cynic = { version = "3.2.2", features = ["rkyv"] }
```

I'd probably pin a specific version to avoid accidental updates.